### PR TITLE
Feature: Add DISCONNECT and KBYE commands

### PR DESCRIPTION
**Pull request by unaveed**
Added the above commands so that the router, workers, and clients can be shutdown gracefully.

### CODE REVIEW

**Comment by unaveed** [comment]
@com4 or @bbtb1982 can one or both of you assign yourself to this pull request? I don't have permission to add assignees.

File: eventmq/conf.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #: SUPER_DEBUG basically enables more debugging logs. Specifically the messages
 #: at different levels in the application.
 #: Default: False
-SUPER_DEBUG = False
+SUPER_DEBUG = True
```
Comment (com4) [inline eventmq/conf.py]: Can you change this back to False before merge it? It's used as the default value.

File: eventmq/router.py [hunk 2]
```diff
@@ -597,6 +627,7 @@ def process_client_message(self, original_msg, depth=0):
             InvalidMessageError: Unable to parse the message
         """
         # Limit recusive depth (timeout on PeerGoneAwayError)
+        logger.info('process_client called orig_msg: {}'.format(original_msg))
```
Comment (com4) [inline eventmq/router.py]: remove this debugging statement

File: eventmq/router.py [hunk 3]
```diff
@@ -704,6 +746,34 @@ def process_worker_message(self, msg):
...
+        Args:
+            worker_id: (str) ID of worker to remove
+        """
+        worker = self.workers.pop(worker_id)
+        for queue in worker['queues']:
+            name = queue[1]
+            workers = self.queues[name]
+            revised_list = filter(lambda x: x[1] != worker, workers)
+            self.queues[name] = revised_list
+
```
Comment (com4) [inline eventmq/router.py]: one last logging comment. can you add a similar log statement in `remove_scheduler` to this method?

**Comment by com4** [comment]
👍

**Comment by com4** [comment]
@unaveed  if you want to fix the conflict i will merge this in. i think it's just your code vs a blank line in jobmanager.py or something.

**Pull request merged**